### PR TITLE
Ruelvla 12

### PR DESCRIPTION
**Pull request by beahuszar**
Sending DB context and setup for pull request. @MichaelBarcley, note that the tables might vary within the database.

### CODE REVIEW

**Comment by beahuszar** [comment]
...int the meantime I continue with the setup of the tables

**Review by MichaelBarcley — COMMENTED** [review]
These changes seems fine for me. Let's wait for Süvi's approval too though.

**Review by drexeverest — APPROVED** [review]
👍

**Pull request merged**
